Public/Src/Tools/UnitTests/DropDaemon/DropOperationTests.cs

```diff
@@ -393,6 +393,101 @@ public void TestAddDirectoryToDropWithFilters(string filter)
                 }).GetAwaiter().GetResult();
         }
 
+        [Theory]
+        [InlineData(true)]
+        [InlineData(false)]
+        public void TestAlwaysFailWhenAddingStaticallyListedFilesWithAbsentFileHash(bool isSourceFile)
+        {
+            var dropPaths = new List<string>();
+
+            var dropClient = new MockDropClient(addFileFunc: (item) =>
+            {
+                dropPaths.Add(item.RelativeDropPath);
+                return Task.FromResult(AddFileResult.Associated);
+            });
+           
+            var ipcProvider = IpcFactory.GetProvider();
+            var bxlApiClient = new Client(ipcProvider.GetClient(ipcProvider.CreateNewConnectionString(), new ClientConfig()));
+
+            WithSetup(
+                dropClient, 
+                (daemon, etwListener) =>
+                {
+                    // only hash and file rewrite count are important here; the rest are just fake values
+                    var hash = FileContentInfo.CreateWithUnknownLength(ContentHashingUtilities.CreateSpecialValue(1)).Render();

```

Comment:
>ContentHashingUtilities.CreateSpecialValue(1) [](start = 71, length = 45)

Is WellKnownHash not accessible? Can it be made accessible?

Reply:
addressed